### PR TITLE
[Monitoring] Inspector

### PR DESCRIPTION
**Pull request by BiliouriV**

### CODE REVIEW

**Review by Salem-Luos — APPROVED** [review]

File: docs/tools/monitoring.md [hunk 1]
```diff
@@ -107,3 +107,38 @@ and send the command:
 Do you need more information on how to debug your application using a sniffer?
 Contact us at <a href="mailto:hello@luos.io">hello@luos.io</a>.
 :::
+
+## Inspector
+
+In order to facilitate the debugging process, Luos offers a more advanced monitoring tool, in order to help you investigate the behavior of your network. In other words, inspector is capable of occupying the information of all the nodes and services in the network, as well as all the messages that are transmitted through it, and finally sending them to any other machine via a serial USB port.
```
Comment (ncros3) [inline docs/tools/monitoring.md:113]: Shouldn't we use "able" instead of "capable" ? 

I don't understand what you mean by "occupying"

**Review by ncros3 — COMMENTED** [review]
I don't have many request but, do you plan to write a tutorial to use the inspector ? I think this could be a good way to show how to use it.

File: docs/tools/monitoring.md (hunk 1 above)
Comment (BiliouriV) [reply docs/tools/monitoring.md]: capable and able are synonims

Comment (ncros3) [reply docs/tools/monitoring.md]: ok.

**Review by ncros3 — APPROVED** [review]

**Pull request merged**